### PR TITLE
DAOS-4058 doc: add properties related stuff in daos manpage

### PR DESCRIPTION
**Pull request by bfaccini**
This is a follow-on to DAOS-3741 changes, to add missing
pool/container "get properties" sub-cmd description, along with
container properties specification during "container create", in
daos command manpage.

Signed-off-by: Bruno Faccini <bruno.faccini@intel.com>

Change-Id: I4f822f57f1725aec26830c3b6ff2c6e247e91e40

### CODE REVIEW

File: doc/man/man8/daos.8 [hunk 1]
```diff
@@ -122,6 +126,21 @@ The \fBRESOURCE\fRs, respective \fBCOMMAND\fRs and \fBOPTION\fRs supported by \f
...
 				      K (KB), M (MB), G (GB), T (TB), P (PB), E (EB)
 .TP
+.I container \fBOPTION\fRs (create):
+	  \fB--properties=\fR<name>:<value>[,<name>:<value>,...]      (optional) container properties
+.br
+				      Supported properties names:
+.br
+				      \fBlabel\fR (can be any string)
+.br
+				      \fBcksum\fR checksum type (can be {off,crc{16,32,64}})
```
Comment (kccain) [inline doc/man/man8/daos.8]: cksum: valid input also includes "sha1" I think (looking at the code)

Comment (bfaccini) [reply doc/man/man8/daos.8]: Nope, still not. See current/master source code using DAOS_PROP_CO_CSUM_SHA1 ... But ok, let's add it already if you want, if it is likely to be supported soon?

**Comment by bfaccini** [comment]
> There are some alignment issues I noticed ...

I am sorry Ken, but I don't see this when I run "man -M doc/man daos" cmd ...

File: doc/man/man8/daos.8 (hunk 1 above)
Comment (kccain) [reply doc/man/man8/daos.8]: I see yes you are right it is not supported yet. But it is in the help output, so perhaps adding it to the manpage now is good to be consistent with that. As I recall early on in the tools proposal we decided the help output should be as complete as possible (given what was understood initially) with placeholders for that known future functionality.

**Comment by kccain** [comment]
> > There are some alignment issues I noticed ...
> 
> I am sorry Ken, but I don't see this when I run "man -M doc/man daos" cmd ...

I see it when I have a terminal that is 80 characters wide. The alignment looks good when I expand the terminal to a wider number of characters.

**Comment by bfaccini** [comment]
Well, looks like all tests have passed, only defect is for optional "Build RPM on Leap 15" step and due to some CI infrastructure related issue, so would be cool if you review my patch so it can land!! ;-)

**Comment by johannlombardi** [comment]
Please port to the 0.9 branch too.

**Pull request merged**